docs/resources/cbr_vault.md

```diff
@@ -66,15 +65,14 @@ resource "huaweicloud_cbr_vault" "test" {
 }
 ```
 
-### Create an SFS turbo type vault
+### Create a SFS turbo type vault
```

Comment:
The letter S is a vowel sound [es] without modification.

Reply:
fixed